### DISCUSSION TITLE
Renogy Smart 100ah not working ... cable issue?

### DISCUSSION THREAD

**Discussion by acarpenter550** [Louisvdw/dbus-serialbattery#962] [General]
Ok been fighting with this for a few hours now and can't get it to work.  I have 2 Renogy 100ah smart batteries in parallel but dbus-serialbattery won't connect to either of them (log shows Incorrect Reply).  I have confirmed  that they are connected to /dev/ttyUSB0 but the driver won't find them at all  (I do see traffic on that TTY port so there is something).  At this point I'm assuming that this is more than likely a cable issue so can someone who has this working with Renogy Smart batteries please post a link to a cable that works?

Currently the cable I'm using is https://www.amazon.com/dp/B082WHR6NW?psc=1&ref=ppx_yo2ov_dt_b_product_details but I also tried https://www.amazon.com/dp/B07BLQ7Q5B?psc=1&ref=ppx_yo2ov_dt_b_product_details.  I have another USB to RS485 adapter (https://www.amazon.com/dp/B078X5H8H7?psc=1&ref=ppx_yo2ov_dt_b_product_details) but I can't find the pinouts to cat5 to make this work.

**Comment by mr-manuel** [comment]
Please post a picture of your cabling.

> I have confirmed that they are connected to /dev/ttyUSB0 

Have you connected two BMS to one adapter? If yes, this won‘t work. You have to connect one BMS to one adapter.

**Comment by acarpenter550** [comment]
Thanks for the response!  The Renogy batteries are interconnected with RS485 so I will try disconnecting them from each other and try my cable with a single battery.

**Comment by acarpenter550** [comment]
No joy with either adapter plugged into the battery ... tried both ports on the battery and both open USB ports on the Cerbo (only have 2 as the 3rd is USB power for the GX display).  I removed the RS485 between the batteries so this should appear as a single battery.  I'm thinking at this point it is either a cable issue or an issue with the firmware version on the battery.

**Reply by sstoops** [reply]
Did you find any resolution? I fought this same issue for awhile before noticing that I was making my RJ45 plug in reverse order. So on a typical ethernet connection (OW, O, GW, B, BW, G, BW, B), B is 1 and OW is 8. I was mistakenly crimping my connection with TX/RX+ on pin 7, TX/RX- on pin 6, and GRND on pin 5.

After realized this issue and made a new connection, my battery showed up in Venus OS immediately.

**Reply by acarpenter550** [reply]
No but I'm thinking it was a cable issue.  I ended up using a different battery in my RV (Chins 200ah) rather than the Renogy but I still have those installed in my truck.  The Chins with the newer bluetooth BMS worked right off the bat.

**Reply by mrnoname22** [reply]
> No but I'm thinking it was a cable issue. I ended up using a different battery in my RV (Chins 200ah) rather than the Renogy but I still have those installed in my truck. The Chins with the newer bluetooth BMS worked right off the bat.

@abomb60 To confirm you were able to make this work with a chins battery that has Bluetooth? Was anything extra needed as far as configuration? Appreciate it!

**Reply by acarpenter550** [reply]
> > No but I'm thinking it was a cable issue. I ended up using a different battery in my RV (Chins 200ah) rather than the Renogy but I still have those installed in my truck. The Chins with the newer bluetooth BMS worked right off the bat.
> 
> @abomb60 To confirm you were able to make this work with a chins battery that has Bluetooth? Was anything extra needed as far as configuration? Appreciate it!

Yes I have this working over Bluetooth to my Cerbo-S in my camper using the CHINS 200ah battery here ... https://www.amazon.com/gp/product/B08N56268P/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1.  Nothing extra was needed.

**Reply by mrnoname22** [reply]
> > > No but I'm thinking it was a cable issue. I ended up using a different battery in my RV (Chins 200ah) rather than the Renogy but I still have those installed in my truck. The Chins with the newer bluetooth BMS worked right off the bat.
> > 
> > 
> > @abomb60 To confirm you were able to make this work with a chins battery that has Bluetooth? Was anything extra needed as far as configuration? Appreciate it!
> 
> Yes I have this working over Bluetooth to my Cerbo-S in my camper using the CHINS 200ah battery here ... https://www.amazon.com/gp/product/B08N56268P/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1. Nothing extra was needed.

Awesome thank you so much for the reply. I ordered the chins 300ah battery and stumbled across this post while looking to see if i could hack the Bluetooth protocol. This is a huge help.

**Reply by mrnoname22** [reply]
> > > No but I'm thinking it was a cable issue. I ended up using a different battery in my RV (Chins 200ah) rather than the Renogy but I still have those installed in my truck. The Chins with the newer bluetooth BMS worked right off the bat.
> > 
> > 
> > @abomb60 To confirm you were able to make this work with a chins battery that has Bluetooth? Was anything extra needed as far as configuration? Appreciate it!
> 
> Yes I have this working over Bluetooth to my Cerbo-S in my camper using the CHINS 200ah battery here ... https://www.amazon.com/gp/product/B08N56268P/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1. Nothing extra was needed.

Would you mind sharing your config.ini? I cannt make mine work and wanted to compare to a working config. I appreciate it!

**Reply by acarpenter550** [reply]
> > > > No but I'm thinking it was a cable issue. I ended up using a different battery in my RV (Chins 200ah) rather than the Renogy but I still have those installed in my truck. The Chins with the newer bluetooth BMS worked right off the bat.
> > > 
> > > 
> > > @abomb60 To confirm you were able to make this work with a chins battery that has Bluetooth? Was anything extra needed as far as configuration? Appreciate it!
> > 
> > 
> > Yes I have this working over Bluetooth to my Cerbo-S in my camper using the CHINS 200ah battery here ... https://www.amazon.com/gp/product/B08N56268P/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1. Nothing extra was needed.
> 
> Would you mind sharing your config.ini? I cannt make mine work and wanted to compare to a working config. I appreciate it!

Unfortunately that is in my camper which is in storage so I won't have access to it until the spring.
